### DISCUSSION TITLE
Cmder + Starship

### DISCUSSION THREAD

**Discussion by maximousblk** [starship/starship#1712] [Q&A] [answered]
Is there a way to use Starship prompt with [Cmder](https://cmder.net)? I use cmder and WSL (Ubuntu) as my daily drivers. While it's fairly straightforward to install Starship on Ubuntu, I couldn't find a way to get it working with Cmder.

So I just wanted to ask, if it is possible (and how) or not?

**Comment by chipbuster** [comment, marked as answer]
I'm not terribly familiar with Cmder, but it looks like it uses Powershell with native windows binaries?

If that's the case, you should be able to download the Windows binaries (https://github.com/starship/starship/releases/download/latest/starship-x86_64-pc-windows-msvc.zip), place them into the `bin` directory specified by the Cmder instructions, and then use the [powershell init instructions](https://starship.rs/#powershell) to enable it in the shell.
